src/CountUp.tsx

```diff
@@ -24,29 +28,31 @@ class CountUp extends Component {
     preserveValue: PropTypes.bool,
   };
 
-  static defaultProps = {
+  static defaultProps: Partial<CountUpProps> = {
     decimal: '.',
     decimals: 0,
-    delay: null,
-    duration: null,
-    easingFn: null,
-    formattingFn: null,
+    duration: 2,
```

Comment:
If we introduce new defaults, we need to make sure to mention it in the release note. 👍

Reply:
@glennreyes I just thought that using defaults from [countUp](https://github.com/inorganik/countUp.js) would be clearer.
https://github.com/inorganik/countUp.js/blob/1e75d958b00fae49badca23a8edfc49c1080de28/src/countUp.ts#L23-L35

Reply:
Makes sense. Let's at least add a comment about the mapping & mention it in the release.